src/poliastro/twobody/orbit.py

```diff
@@ -389,30 +389,37 @@ def from_body_ephem(cls, body, epoch=None):
                 TimeScaleWarning,
             )
 
-        r, v = get_body_barycentric_posvel(body.name, epoch)
-
-        if body == Moon:
-            # TODO: The attractor is in fact the Earth-Moon Barycenter
-            icrs_cart = r.with_differentials(v.represent_as(CartesianDifferential))
-            gcrs_cart = (
-                ICRS(icrs_cart)
-                .transform_to(GCRS(obstime=epoch))
-                .represent_as(CartesianRepresentation)
-            )
-            ss = cls.from_vectors(
-                Earth,
-                gcrs_cart.xyz.to(u.km),
-                gcrs_cart.differentials["s"].d_xyz.to(u.km / u.day),
-                epoch,
-            )
+        try:
+            r, v = get_body_barycentric_posvel(body.name, epoch)
+            if body == Moon:
+                # TODO: The attractor is in fact the Earth-Moon Barycenter
+                icrs_cart = r.with_differentials(v.represent_as(CartesianDifferential))
+                gcrs_cart = (
+                    ICRS(icrs_cart)
+                    .transform_to(GCRS(obstime=epoch))
+                    .represent_as(CartesianRepresentation)
+                )
+                ss = cls.from_vectors(
+                    Earth,
+                    gcrs_cart.xyz.to(u.km),
+                    gcrs_cart.differentials["s"].d_xyz.to(u.km / u.day),
+                    epoch,
+                )
 
-        else:
-            # TODO: The attractor is not really the Sun, but the Solar System
-            # Barycenter
-            ss = cls.from_vectors(Sun, r.xyz.to(u.km), v.xyz.to(u.km / u.day), epoch)
-            ss._frame = ICRS()  # Hack!
+            else:
+                # TODO: The attractor is not really the Sun, but the Solar System
+                # Barycenter
+                ss = cls.from_vectors(
+                    Sun, r.xyz.to(u.km), v.xyz.to(u.km / u.day), epoch
+                )
+                ss._frame = ICRS()  # Hack!
 
-        return ss
+            return ss
+
+        except KeyError:
+            raise ValueError(
+                "Pluto cannot be plotted using the builtin ephemeris, to change the builtin ephemeris please add the following lines to your code, and set the ephemeris according to your needs: \n\nfrom astropy.coordinates import solar_system_ephemeris\nsolar_system_ephemeris.set('jpl')"
```

Comment:
It's not a problem of `Pluto` only (this message can be raise with the Moon as well, as far as I understand) and it's not a problem of plotting.

Reply:
And in fact, you can just move this message to here:

https://github.com/poliastro/poliastro/blob/15986401bf3d745c378303dccdc8ebfe77e1a14d/src/poliastro/threebody/soi.py#L55-L64